rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -969,68 +974,91 @@ void registerBtcTransaction_whenValidationPeriodEnded_shouldNotProcessSpendTx()
             bridgeStorageProvider.save();
 
             // assert
+            // tx was not registered
             assertActiveFederationUtxosSize(activeFederationUtxosSizeBeforeRegisteringTx);
             assertTransactionWasNotProcessed(svpSpendTransaction.getHash());
+
+            // svp success was not processed
             assertSvpSpendTxHashUnsignedIsSavedInStorage();
+            assertNoHandoverToNewFederation();
+            assertProposedFederation();
         }
 
         @Test
-        void registerBtcTransaction_whenIsTheSpendTransaction_shouldProcessSpendTx() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
+        void registerBtcTransaction_whenIsNotTheSpendTransaction_shouldNotProcessSpendTx() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
```

Comment:
Should we add the test cases when tx is a PEGIN or UNKNOWN also?

Reply:
the pegin case could be, the unknown i dont think is necessary.
but we definitely should be testing that the spend tx is being processed even when it does not have the minimum pegin value! So thanks

Reply:
This is interesting, if the spend tx is below the min peg-in value I don't think the powpeg will inform it to the Bridge